pyfolio/plotting.py

```diff
@@ -1005,7 +938,7 @@ def plot_rolling_volatility(returns, factor_returns=None,
 
     ax.set_ylabel('Volatility')
     ax.set_xlabel('')
-    if factor_returns.empty:
```

Comment:
@ssanderson I'm not sure I see how this function would crash if `factor_returns` was `None`. Could you point out a line number?

Reply:
On this line, we're accessing `factor_returns.empty`, and `NoneType` doesn't have an `empty` attribute.

Reply:
Yes, which is why I changed it to `if factor_returns is None`; there's no more reference to the `empty` attribute` anywhere in this function. Am I missing something?

Reply:
@eigenfoo my point here was that, since `factor_returns` was previously required for this function to work, we might want to just consider making it actually a required argument and not call this function at all if we don't have a benchmark. Whether or not that makes sense depends on how useful we think this plot is without access to the benchmark.

Reply:
Ah I see! No, that was intentional: see [this issue](https://github.com/quantopian/pyfolio/issues/534). The idea is that we want to make pyfolio as benchmark-independent as possible: if a benchmark is passed, everything works as required. If not, pyfolio simply skips the analyses that depend on benchmarks.